### DISCUSSION TITLE
Discord oAuth support

### DISCUSSION THREAD

**Discussion by NekrosBurek** [pelican-dev/panel#121] [Ideas]
Hello everyone,

Since Discord is widely used and for some even day to day application, i would suggest option to allow registration and logging in with Discord oAuth, with this it would make it easier to add user as subuser or just create a server for user.

**Comment by tieb62** [comment]
I think more generic OpenID support would be better, as explained in #173

**Comment by lancepioch** [comment]
Yup, Discord and Oauth support is planned, thanks!
